### DISCUSSION TITLE
DML in Bau

### DISCUSSION THREAD

**Discussion by artydev** [grucloud/bau#34] [Ideas]
Hy Frederic,

Thank for your amazing job :-)
Would you consider to add [DML](https://github.com/vanjs-org/van/tree/main/addons/van_dml) in BAU ?

Regards

**Comment by FredericHeem** [comment]
Adding 2 different way to describe the ui creates more complexity and fragmentation. 
Moreover, this DML syntax makes auto formatting impossible, doesn't it ?  
However, feel free to create a separate repo to implement DML on top of Bau.
